api.ts

```diff
@@ -2185,11 +2189,19 @@ export interface SyncTableOptions<
   PermissionsContextT extends SyncPassthroughData,
 > {
   /**
-   * The name of the sync table. This is shown to users in the Coda UI.
+   * The name of the sync table. This is shown to users in the Coda UI if displayName is not present.
    * This should describe the entities being synced. For example, a sync table that syncs products
    * from an e-commerce platform should be called 'Products'. This name must not contain spaces.
+   *
+   * Important: This should not be updated after the first time a table is included in the pack version.
```

Comment:
Maybe better to reframe this as a breaking change.

"Important: This value acts as a unique ID for the table, and updating it later is a breaking change. If you want to change the value shown to the users set `displayName` instead.